src/Field/NominatimSearch/NominatimSearch.jsx

```diff
@@ -258,16 +258,6 @@ export class NominatimSearch extends React.Component {
   render() {
     const {
       className,
-      nominatimBaseUrl,
-      format,
-      viewbox,
-      bounded,
-      polygon_geojson,
-      addressdetails,
-      limit,
-      countrycodes,
-      map,
-      onSelect,
```

Comment:
huh? is this ok here?

Reply:
I think this is mistake… plz revert.

Reply:
I don't see where these variables are used in `render` function?!

Reply:
They are not used but you have to extract them from the `passThroughProps` as these props would be all passed to antds `AutoComplete` which does not accept them.

Reply:
reverted in 0e724b2fbac1e89